pwa/src/components/endpoints/endpointForm.tsx

```diff
@@ -230,47 +207,50 @@ export const EndpointForm: React.FC<EndpointFormProps> = ({id}) => {
-                                      nameOverride={"Applications"}
-                                      value={"/admin/applications/"}/>
-                                  ) : (
-                                    <SelectInputComponent
-                                      options={applications}
-                                      name={"application"}
-                                      id={"applicationInput"}
-                                      nameOverride={"Applications"}
-                                      value={"/admin/applications/"}/>
-                                  )}
-                                </>
-                              ) : (
-                                <SelectInputComponent
-                                  options={[{name: "Please create a Application.", value: null}]}
-                                  name={"application"}
-                                  id={"applicationInput"}
-                                  nameOverride={"Applications"}/>
-                              )}
+                            {applications !== null && applications.length > 0 ? (
+                              <>
+                                {endpoint !== null &&
+                                endpoint.application !== undefined &&
+                                endpoint.application !== null ? (
+                                  <SelectInputComponent
+                                    options={applications}
+                                    data={endpoint.application.name}
+                                    name={"application"}
+                                    id={"applicationInput"}
+                                    nameOverride={"Applications"}
+                                    value={"/admin/applications/"}
+                                  />
+                                ) : (
+                                  <SelectInputComponent
+                                    options={applications}
+                                    name={"application"}
+                                    id={"applicationInput"}
+                                    nameOverride={"Applications"}
+                                    value={"/admin/applications/"}
+                                  />
+                                )}
+                              </>
+                            ) : (
+                              <SelectInputComponent
+                                options={[{name: "Please create a Application.", value: null}]}
+                                name={"application"}
+                                id={"applicationInput"}
+                                nameOverride={"Applications"}
+                              />
+                            )}
                           </div>
                         </div>
                       </div>
+                      )
                     </div>
                   )}
```

Comment:
You've done some cleanup in this file, which is good. However, during this, I think you added a `)` too much somewhere. Resulting in this `)` actually rendering to the DOM, see screenshot:

![image](https://user-images.githubusercontent.com/66552932/152522367-4f446811-3be1-402b-aa13-f09f86058c67.png)